pkg/scheduler/core/generic_scheduler.go

```diff
@@ -227,6 +228,7 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 	metrics.SchedulingLatency.WithLabelValues(metrics.PredicateEvaluation).Observe(metrics.SinceInSeconds(startPredicateEvalTime))
 	metrics.DeprecatedSchedulingLatency.WithLabelValues(metrics.PredicateEvaluation).Observe(metrics.SinceInSeconds(startPredicateEvalTime))
 
+	klog.V(2).Infof("DEBUG: Prioritizing pod: %v/%v/%v", pod.Tenant, pod.Namespace, pod.Name)
```

Comment:
Perhaps use tracing instead?

Reply:
this is intentional to just log it so one can just use log entry time stamp to align the pod activities in the scheduler which cross multiple components.  a scheduler-scope tracing object might work but it will be complicated with multiple pods.
note that the tracing is already in there but only logs out for pod that scheduling time greater than 100ms. 

also those log entry probably will not be merge into production so it is marked as "DEBUG"